include/dav1d/picture.rs

```diff
@@ -290,10 +294,24 @@ pub(crate) struct Rav1dPicAllocator {
     pub cookie: *mut c_void,
 
     /// See [`Dav1dPicAllocator::alloc_picture_callback`].
+    ///
+    /// # Safety
+    ///
+    /// If frame threading is used, accesses to [`Self::cookie`] must be thread-safe.
+    /// If [`Self::cookie`] is accessed in Rust and has interior mutability, it must go through [`UnsafeCell`].
```

Comment:
I don't think mentioning UnsafeCell here is necessary or helpful (most Rust usage of UnsafeCell is encapsulated in other types anyway).

Reply:
As discussed, I'll replace this with saying the cookie needs to be `Send + Sync`.